containers/Dockerfile.access-om3

```diff
@@ -0,0 +1,24 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+ARG PACKAGE_NAMES="access-om3"
```

Comment:
To be consistent with `access-om2` we should use the model components as defined in here

https://github.com/COSIMA/access-om3/tree/master

Namely: CDEPS, CICE6, CMEPS, MOM6, WW3

But we don't have package files for any of those so we can't put them in there. So `access-om3` is a decent place-holder as it just defines a few dependencies.

Reply:
Noted, will keep in mind when we add package files